core/pom.xml

```diff
@@ -189,18 +189,6 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
-        <dependency>
```

Comment:
Just curious @coheigea if we should use this dependency vs shaded one. It seems like Woodstox uses this library anyway (https://github.com/FasterXML/woodstox/blob/df8d1648cd8bf5c4e622f00156d62ef3fd88d7c5/pom.xml#L42), may be we could keep it the old way (plus, it would stay optional). What do you think?

Reply:
I see, thanks @coheigea